integration-tests/utils/integration-utils.js

```diff
@@ -86,7 +86,7 @@ const dismissNotifications = async page => {
 const signIntoTerra = async (page, token = bearerToken) => {
   await page.waitForXPath('//*[contains(normalize-space(.),"Loading Terra")]', { hidden: true })
   await waitForNoSpinners(page)
-  await page.evaluate(token => window.forceSignIn(token), token)
+  await page.evaluate(async token => await window.forceSignIn(token), token)
```

Comment:
This seemed like a good theory, but I'm not sure anymore. I attempted to test it by inserting delays during `forceSignIn` and could not reproduce the failure. Looking more closely at `forceSignIn`:
```js
window.forceSignIn = async token => {
  const res = await fetch(
    'https://www.googleapis.com/oauth2/v3/userinfo',
    { headers: { Authorization: `Bearer ${token}` } }
  )
  const data = await res.json()
  authStore.update(state => {
    return {
      ...state,
      isSignedIn: true,
      registrationStatus: undefined,
      isTimeoutEnabled: undefined,
      profile: {},
      user: {
        token,
        id: data.sub,
        email: data.email,
        name: data.name,
        givenName: data.given_name,
        familyName: data.family_name,
        imageUrl: data.picture
      }
    }
  })
}
```
The registration form isn't going to render until the sign-in state changes. Both the sign-in state and the email address that is pre-populated into the form are updated at the same time. I'm starting to wonder if we're not getting a success response from `userinfo`. In fact, the way that I found to reproduce the failure behavior was to give `userinfo` a bad token so I would definitely get an error response.

Reply:
Thanks. I spoke with Isaac to clear up some confusion I had regarding page.evaluate as well.
I opted to add error reporting to this function so we can get better visibility into the issue when it happens again